docs/guides/add-onboarding-flow.mdx

```diff
@@ -171,7 +174,8 @@ export default function OnboardingComponent() {
   const handleSubmit = async (formData: FormData) => {
     const res = await completeOnboarding(formData);
     if (res?.message) {
-      await user?.reload();
+      // Call reload() to refresh the user's session token 
```

Comment:
Not sure this comment is necessary with the new Callout section
```suggestion
```

Reply:
I'd prefer a comment than a callout

Reply:
Done!